README.md

```diff
@@ -95,8 +95,15 @@ orchestrator.registerScenario('proper zome call', async (s, t) => {
 
   // and install a single happ
   const carol_blog_happ = await carol.installHapp([dnaBlog])
-  // or a happ with a previously generated key
-  const carol_test_happ_with_bobs_test_key = await carol.installHapp([dnaTest], bob_blog_happ.agent)
+
+  // or a happ with a previously generated key and previously registered dna by hash
+  // which you can get from the cells as returned in the InstalledHapp from previous install
+  const testDnaHash = bob_test_happ.cells[0].dnaHash()
+  const carol_test_happ_with_bobs_test_key = await carol.installHapp([testDnaHash], bob_blog_happ.agent)
+
+  // or from a call to registerDna which might look like this to create a derived dna using
+  // different uuid.
+  const derivedDnaHash = await carol.registerDna({hash: testDnaHash}, "1234567890")
```

Comment:
I don't fully understand. Does this mean that if you install a DNA with one conductor, then another conductor on the same machine has access to that DNA and will attempt to look for it whenever it receives the DNA hash? I was assuming that each conductor running on the same machine has entirely separate databases.

Reply:
oops, bad example.  They are separate.  I'll fix.